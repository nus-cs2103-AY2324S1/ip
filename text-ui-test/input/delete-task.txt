todo do some stuff quack
event hello /from 8pm tday /to 8pm tmr
deadline finish some stuff /by this Sunday
list
delete 4
list
delete 1
list
mark 2
delete 2
list
bye