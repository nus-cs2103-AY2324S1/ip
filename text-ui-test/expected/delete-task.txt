____________________________________________________________
Loading data from hard disk ...
Done loading.
____________________________________________________________
Hello from Quack-NKN
What can I do for you?
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[T][ ] do some stuff quack
Now you have 1 in the list.
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[E][ ] hello (from: 8pm tday to: 8pm tmr)
Now you have 2 in the list.
____________________________________________________________
In: ____________________________________________________________
Got it. I've added this task:
[D][ ] finish some stuff (by: this Sunday)
Now you have 3 in the list.
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do some stuff quack
2. [E][ ] hello (from: 8pm tday to: 8pm tmr)
3. [D][ ] finish some stuff (by: this Sunday)
____________________________________________________________
In: ____________________________________________________________
Quack, the task number you provide does not exist!
____________________________________________________________
In: ____________________________________________________________
1. [T][ ] do some stuff quack
2. [E][ ] hello (from: 8pm tday to: 8pm tmr)
3. [D][ ] finish some stuff (by: this Sunday)
____________________________________________________________
In: ____________________________________________________________
Noted, I've removed this task:
[T][ ] do some stuff quack
Now you have 2 in the list.
____________________________________________________________
In: ____________________________________________________________
1. [E][ ] hello (from: 8pm tday to: 8pm tmr)
2. [D][ ] finish some stuff (by: this Sunday)
____________________________________________________________
In: ____________________________________________________________
Nice! I've marked this task as done:
[D][X] finish some stuff (by: this Sunday)
____________________________________________________________
In: ____________________________________________________________
Noted, I've removed this task:
[D][X] finish some stuff (by: this Sunday)
Now you have 1 in the list.
____________________________________________________________
In: ____________________________________________________________
1. [E][ ] hello (from: 8pm tday to: 8pm tmr)
____________________________________________________________
In: ____________________________________________________________
Bye. Hope to see you again soon!
____________________________________________________________